torchbenchmark/models/mobilenet_v2_quantized_qat/__init__.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, device=None, jit=False):
         self.model = models.mobilenet_v2().to(self.device)
         self.example_inputs = (torch.randn((96, 3, 224, 224)).to(self.device),)
         self.prep_qat_train()  # config+prepare steps are required for both train and eval
+        torch.cuda.empty_cache()
```

Comment:
hmm... makes me wonder why this helps?  i think if the cache can be emptied, the space would be reused automatically rather than throw oom.  maybe there is something wrong with our allocator?

Reply:
I was trying to use this to fix the OOM error but looks like it doesnot work. I am trying other methods to fix this.

Reply:
Should be fixed by using `gpu.nvidia.medium` runner.